src/js/views/patients/sidebar/patient/patient-sidebar.scss

```diff
@@ -0,0 +1,21 @@
+.patient-sidebar__user-icon svg,
+.patient-sidebar__close-icon svg {
+  font-size: 20px !important;
+}
+
+.patient-sidebar__patient-info {
+  border-bottom: 1px solid $black-90;
+  padding: 16px 0;
+}
+
+.patient-sidebar__patient-name {
```

Comment:
we have a naming collision on this file where the block `patient-sidebar` is already defined.  such that this class and `patient-sidebar__name` are a little confusing.  In fact since we are using `patient-sidebar__section` with this technically we are using that other BEM'd sass file.  I think we should make a `sass/domain/patient-sidebar.scss` and move `patient-sidebar__section` to it and include it in both view files.  Then perhaps we can rename these class names slightly to something like `worklist-patient-sidebar__` to avoid any future accidental collision.

Reply:
Ya that is confusing, will have that new `sass/domain/patient-sidebar.scss` file created and the class names changed to `worklist-patient-sidebar__`.